### ISSUE TITLE
Should /results redirect anywhere?

### ISSUE THREAD

**Issue by jfly** [jfly/ccm#119]
Right now it just shows a list of events. Should we do something clever instead? I can't think of anything too useful here: redirect to the most recently changed round? Redirect to an open round at random? Only redirect to a round if it's the *only* open round?

@jbcm627, any ideas?

**Comment by jfly** [comment]
The place to put any logic for this is in ViewRoundController's data method in routes.js.

**Comment by jbcm627** [comment]
So right now nothing loads, meaning everyone has to click an extra link to get to a particular round.  We can save at least some people from needing to do this; I don't have anything concrete to back this up, but I think most people will be attempting to see the latest round with results in it, so I would vote for loading that page.

**Comment by FatBoyXPC** [comment]
Maybe we can pop in the results as they get entered? Show the event + competitor + time(s)? It'd be like a feed, essentially.

**Comment by jfly** [comment]
Oh I LIKE. Perhaps a ticker tape style thing?

**Comment by FatBoyXPC** [comment]
Yeah, I was thinking something like the newest result just pushes the
others down. Obviously we'd have like a 20 line feed or something and start
popping the items out of the content view, as well.

On Fri, Jan 23, 2015 at 3:08 AM, Jeremy Fleischman <notifications@github.com
